File: cli/src/test/resources/testing/connecteds/ConnectedsConnected.java
package connecteds;

import org.roaringbitmap.RoaringBitmap;

class ConnectedsConnected {
    public static Object newBitsetConnected() {
        return new RoaringBitmap();
    }

    public static void bitsetAdd(Object bitset, int i) {
        ((RoaringBitmap)bitset).add(i);
    }

    public static boolean bitsetContains(Object bitset, int i) {
        return ((RoaringBitmap)bitset).contains(i);
    }
}

File: cli/src/test/resources/testing/connecteds/ConnectedsConnected.cs
using R = Roaring.Net.CRoaring;

namespace Connecteds
{
    static class ConnectedsConnected
    {
        internal static object NewBitsetConnected()
        {
            return new R::Roaring32Bitmap();
        }

        internal static void BitsetAdd(object bitset, int i)
        {
            // Two's complement is fine here, providing unique values.
            ((R::Roaring32Bitmap)bitset).Add(unchecked((uint)i));
        }

        internal static bool BitsetContains(object bitset, int i)
        {
            // Two's complement is fine here, providing unique values.
            return ((R::Roaring32Bitmap)bitset).Contains(unchecked((uint)i));
        }
    }
}
